source/administration/extended-support-release.rst

```diff
@@ -27,19 +31,34 @@ If your organization prefers to have the newest features and improvements, Exten
++-------------+----------------+------------------+------------------+--------------------------------------------------------------------------------------------+-------------------------------------------------+
+| 5.19        | Quality        | January 16, 2020 | October 15, 2020 |                                                                                            |                                                 |
++-------------+----------------+------------------+------------------+--------------------------------------------------------------------------------------------+-------------------------------------------------+
+| 5.9         | Quality        | April 16, 2019   | April 15, 2020   | `5.9.6 <https://releases.mattermost.com/5.9.6/mattermost-5.9.6-linux-amd64.tar.gz>`_       | Please upgrade to 5.0 prior to upgrading to 5.9 |
++-------------+----------------+------------------+------------------+--------------------------------------------------------------------------------------------+-------------------------------------------------+
+| 4.10        | Quality        | May 16, 2018     | July 15, 2019    | `4.10.10 <https://releases.mattermost.com/4.10.10/mattermost-4.10.10-linux-amd64.tar.gz>`_ |                                                 |
++-------------+----------------+------------------+------------------+--------------------------------------------------------------------------------------------+-------------------------------------------------+
 
 How do I restore a previous Extended Support Release?
 -----------------------------------------------------
-Before you perform an upgrade, please ensure you have done a full back up of your current version.  You can restore the database and previous version if you need to revert an upgrade.  Please note that previous ESR versions are subject to an end of support date. 
+Before you perform an upgrade, please ensure you have done a full back up of your current version.  You can restore the database and previous version if you need to revert an upgrade.  Please note that previous ESR versions are subject to an end of support date.
+
+Why is an Extended Support Release supported for 9 months and not longer?
+-------------------------------------------------------------------------
+We chose 9 months because the overhead for maintaining an ESR is high (and becomes higher the longer the support window is).
+If we increase the support window, it decreases how much we can develop the product. We specifically chose "ESR" instead of "LTS", since it is not intended to be a multi-year longterm support release. The 9 months support timeframe will not be extended in any case.
+
+Can customers pay for extended support?
+---------------------------------------
+At this point, we are not planning on letting customers pay for extended support, but we are open to discuss options for this.
```

Comment:
0/5 Maybe we indicate something along the lines of "Please speak to your Customer Success Manager if you have additional requirements for extended support" so they know their path of escalation?

Reply:
Thanks! Fixed,